File: Javascript/eliminacion/eliminacion.js
// Programa para eliminar un elemento al principio de un array
let inputArr = [11, 21, 31, 41, 51, 61];
console.log("Antes de la eliminacion, el array es: ");
process.stdout.write(inputArr.join(" "));

// Eliminando el primer elemento del inputArr[0]
console.log("\nDespues de la eliminacion, el array es: ");
inputArr.shift();
process.stdout.write(inputArr.join(" "));

// Eliminando el ultimo elemento del inputArr[-1]
inputArr = [11, 21, 31, 41, 51, 61];
console.log("\nDespues de la eliminacion, el array es: ");
inputArr.pop();
process.stdout.write(inputArr.join(" "));

// Eliminando el numero del medio el inputArr[2]
inputArr = [11, 21, 31, 41, 51, 61];
console.log("\nDespues de la eliminacion, el array es: ");
inputArr.splice(2, 1);
process.stdout.write(inputArr.join(" "));
console.log();

File: C++/eliminacion/Eliminacion.cpp
#include <iostream>
#include <vector>
using namespace std;

// Programa para eliminar un elemento al principio de un array
int main()
{
    vector<int> inputArr = {11, 21, 31, 41, 51, 61};
    cout << "Antes de la eliminacion, el array es: " << endl;
    for (size_t j = 0; j < inputArr.size(); j++)
    {
        cout << inputArr[j] << " ";
    }

    // Eliminando el primer elemento del inputArr[0]
    cout << "\nDespues de la eliminacion, el array es: " << endl;
    inputArr.erase(inputArr.begin());
    for (size_t j = 0; j < inputArr.size(); j++)
    {
        cout << inputArr[j] << " ";
    }

    // Eliminando el ultimo elemento del inputArr[-1]
    inputArr = {11, 21, 31, 41, 51, 61};
    cout << "\nDespues de la eliminacion, el array es: " << endl;
    inputArr.erase(inputArr.end() - 1);
    for (size_t j = 0; j < inputArr.size(); j++)
    {
        cout << inputArr[j] << " ";
    }

    // Eliminando el numero del medio el inputArr[2]
    inputArr = {11, 21, 31, 41, 51, 61};
    cout << "\nDespues de la eliminacion, el array es: " << endl;
    inputArr.erase(inputArr.begin() + 2);
    for (size_t j = 0; j < inputArr.size(); j++)
    {
        cout << inputArr[j] << " ";
    }

    return 0;
}
